src/Hangfire.SqlServer/SqlServerConnection.cs

```diff
@@ -341,6 +341,26 @@ public override int RemoveTimedOutServers(TimeSpan timeOut)
                 new { timeOutAt = DateTime.UtcNow.Add(timeOut.Negate()) }));
         }
 
+        public override List<string> FetchAndRemoveTimedOutServers(TimeSpan timeOut)
+        {
+            if (timeOut.Duration() != timeOut)
+            {
+                throw new ArgumentException("The `timeOut` value must be positive.", "timeOut");
+            }
+
+            string query = $@"select Id from [{nameof(_storage)}].Server where LastHeartbeat < @timeOutAt";
```

Comment:
string is used for all the sql queries in the file, a bit inconsistent to change one method to var.